### DISCUSSION TITLE
Eclipse: Failed to execute MI command -exec-run

### DISCUSSION THREAD

**Discussion by ATS-data** [eclipse-cdt/cdt#489] [Q&A]
I have a piezo from Mad City Labs, and I'm trying to compile the example provided with their C API. I've used a different IDE before and it works, but now I need to do it in Eclipse.

I've added the .lib file path to the path variables.
![linked resources](https://github.com/eclipse-cdt/cdt/assets/139219413/3bef65f9-75ae-45d1-816d-b434ad1ace50)

The program builds.

If I run, I get this message:
<terminated> (exit value: -1.073.741.515) (...)

But when I try to debug, I get this error:
![error](https://github.com/eclipse-cdt/cdt/assets/139219413/24d727bd-6b14-4763-b0a5-52aa71fd23ff)

What should I do?

(I've posted on Eclipse forum and the only answer I got was an advice to post here)

**Comment by jonahgraham** [comment]
Thanks for the question. This is a known regression due to a change in gdb/Cygwin in recent years. Please see this issue for details https://github.com/eclipse-cdt/cdt/issues/228#issuecomment-1456858834
